src/incremental/serialize.ml

```diff
@@ -7,7 +7,7 @@ let cil_file_name = "ast.data"
 let solver_data_file_name = "solver.data"
 let results_dir = "results"
 let results_tmp_dir = "results_tmp"
-
+let heap_file_name = "heap.data"
```

Comment:
Now that mallocWrapper hashtables are marshaled via `init`/`finalize`, I guess this isn't used anywhere anymore?

Reply:
Yes, thanks for the hint. Must have missed this when merging.